warning  File ignored because of a matching ignore pattern

When I run it I get these warnings:

```
/home/ubuntu/app/dist/web/vnd.timers-browserify.js
  0:0  warning  File ignored because of a matching ignore pattern. Use "--no-ignore" to override
```

It seems to be applying the rules from `.eslintignore`.  It might be nice if I could use `--no-ignore` as suggested, but that option is not propagated through by the command line tool.

--- a/packages/check-es-compat/bin/cli.js
+++ b/packages/check-es-compat/bin/cli.js
@@ -19,7 +19,10 @@ if (args.length === 0) {
 
 function execute(files) {
   const eslintCLI = new eslint.CLIEngine({
-    useEslintrc: false, // ignore any config files
+    // Ignore any config files
+    useEslintrc: false,
+    ignore: false,
+
     plugins: ['ecmascript-compat'],
     parserOptions: {
       // Latest version, so all features work

